### PR TITLE
Bash script and versions corrected; fixes #4

### LINKED ISSUE #4: Create bash script to mirror compose.bat functionality.
To enable good execution on Linux systems, a bash script has to be made to mirror the functionality of `compose.bat`.

### PR DESCRIPTION
**Pull request by alexatrhea**
The bashscript to mirror `compose` is added.

@adrianchivu or @cozminvelciu would you be so kind as to verify on ubuntu that they work?

### CODE REVIEW

**Comment by alexatrhea** [comment]
> Tested on Ubuntu Bionic 18.04 with docker-compose version 1.26.0 and I can confirm that it works properly.
> On small remark: I've added execute permission to the compose.sh script.

where did you add the permissions?

**Comment by adrianchivu** [comment]
> > Tested on Ubuntu Bionic 18.04 with docker-compose version 1.26.0 and I can confirm that it works properly.
> > On small remark: I've added execute permission to the compose.sh script.
> 
> where did you add the permissions?

To the compose.sh shell script because cannot be executed.

**Comment by alexatrhea** [comment]
> > > Tested on Ubuntu Bionic 18.04 with docker-compose version 1.26.0 and I can confirm that it works properly.
> > > On small remark: I've added execute permission to the compose.sh script.
> > 
> > 
> > where did you add the permissions?
> 
> To the compose.sh shell script because cannot be executed.

But you just mean you had to run `chmod` before running right? Nothing required in code.

**Comment by cozminvelciu** [comment]
> 
> 
> > > > Tested on Ubuntu Bionic 18.04 with docker-compose version 1.26.0 and I can confirm that it works properly.
> > > > On small remark: I've added execute permission to the compose.sh script.
> > > 
> > > 
> > > where did you add the permissions?
> > 
> > 
> > To the compose.sh shell script because cannot be executed.
> 
> But you just mean you had to run `chmod` before running right? Nothing required in code. Correct, before running, nothing required into the code.

I think what he meant is that after running `chmod`, git detects the file as changed and should be commited. Afaik git is a bit weird with file permissions on Windows filesystems, but the changes seem to be seen properly on Linux.

See [git config core.fileMode](https://mirrors.edge.kernel.org/pub/software/scm/git/docs/git-config.html).

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This is the Git repository of the official `CDP4 Community Edition` docker compose scripts. The scripts have been designed in a way that you can spin up various combinations of the CDP4 database and webservices for production,development as well as testing.
 
+## Compatibility
+
+**If you are going to create own variations on these docker-compose files or create your own bear in mind that version `4.0.0` of CDP Webservices does not support database images version higher than `1.3.1`.**
```
Comment (cozminvelciu) [inline README.md:7]:
```suggestion
**If you are going to create your own variations on these docker-compose files or create your own, bear in mind that version `4.0.0` of CDP Webservices does not support database images version higher than `1.3.1`.**
```

File: compose.sh [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+start() {
+	docker-compose start
+}
+
+stop() {
+	docker-compose stop
+}
+
+up() {
+	docker-compose up -d
```
Comment (cozminvelciu) [inline compose.sh:19]: In the windows script, `up` also does a `docker-compose down --remove-orphans` as the first command.
This is also the same command run by `down`, maybe in both scripts we could just call inside `up` the `down` function as the first thing?

File: docker-compose-test.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 version: '3.8'
 services:
   cdp4_db:
-    image: rheagroup/cdp4-test-database-community-edition:latest
+    image: rheagroup/cdp4-test-database-community-edition:1.3.1
```
Comment (cozminvelciu) [inline docker-compose-test.yml:4]: Why is the move needed from the `latest` tag to hardcoded version?

Comment (alexatrhea) [reply docker-compose-test.yml:4]: This is temporary. DockerHub currently does not have `latest` yet. It was added in mind for the upcoming release.

**Pull request merged**